aws/cloudformation/standalone/marketing/deployment/marketing.yml.erb

```diff
@@ -507,6 +514,24 @@ Resources:
     UpdateReplacePolicy: Delete
     DeletionPolicy: Delete
 
+  MarketingStaticAssetsBucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketName: !Sub "cdo-marketing-static-assets-${Environment}"
+      PublicAccessBlockConfiguration:
+        BlockPublicAcls: true
+        BlockPublicPolicy: true
+        IgnorePublicAcls: true
+        RestrictPublicBuckets: true
+      VersioningConfiguration:
+        Status: Enabled
+      LifecycleConfiguration:
+        Rules:
+          - Id: ExpireOldVersions
+            Status: Enabled
+            NoncurrentVersionExpiration:
+              NoncurrentDays: 90
+
```

Comment:
Let’s also enable S3 Intelligent Tiering so that old assets that are not being retrieved frequently are automatically moved to cheaper storage (though still accessible with millisecond response time):
```yaml
    IntelligentTieringConfigurations:
      - Id: EntireBucket
        Status: Enabled
```